main/src/main/resources/org/clulab/numeric/atomic.yml

```diff
@@ -24,6 +24,14 @@ rules:
     pattern: |
       [word=/^(1\d\d\d|20\d\d)$/]
 
+  # weak possible years: 1d, 2d, 3d, 4d, 5d, 6d, 7d, 8d, 9d
+  - name: weakyear
+    label: WeakPossibleYear
+    priority: ${ rulepriority }
+    type: token
+    pattern: |
+      [word=/^(1\d|2\d|3\d|4\d|5\d|6\d|7\d|8\d|9\d)$/]
```

Comment:
I wonder why this isn't `[1-9]\d`.

Reply:
Yes, it should be. It started as just [1-2] and [8-9] then I incrementally expanded without optimizing. Please adjust!